### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
First release of jobinfo. With a fix for handling missing data for memory information and improved README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,26 @@ Tool for giving a quick overview of Slurm job statistics.
...
+collates job information from the 'sstat', 'sacct' and 'squeue' SLURM commands
+to give a uniform interface for both current and historical jobs.
+
+positional arguments:
+  jobid        the jobid to query
+
+optional arguments:
+  -h, --help   show this help message and exit
+  -d, --debug  Shows extra information about how the data is gathered
+  -l, --long   Shows more information about the job
```
Comment (bedroge) [inline README.md]: Nitpicking, but let's make this consistent:
```suggestion
  -h, --help   show this help message and exit
  -d, --debug  show extra information about how the data is gathered
  -l, --long   show more information about the job
```

File: README.md [hunk 2]
```diff
@@ -3,3 +3,26 @@ Tool for giving a quick overview of Slurm job statistics.
...
+```
+
+# Release notes:
+
+## v1.0 - 2024-04-15
```
Comment (bedroge) [inline README.md]: Shouldn't this be May?

**Comment by bedroge** [comment]
Don't know if I was supposed to review all code or just the changes from this PR, but I mostly did the latter, and also tried the new jobinfo on some jobs myself. This worked fine, only ran into one thing that looked a bit odd for job 8916761:
```
State                          : CANCELLED,COMPLETED,TIMEOUT  
```

By using sacct I found that this is actually a combination of all statuses (job, batch, step), and now it does make a bit more sense to me. I don't really know what it should report in such cases anyway, as it may always be a bit confusing. The old jobinfo reports:
```
State               : CANCELLED,TIMEOUT
```
:man_shrugging:

File: README.md (hunk 1 above)
Comment (FokkeDijkstra) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (FokkeDijkstra) [reply README.md]: Thanks for spotting this. Fixed.

**Comment by FokkeDijkstra** [comment]
Thanks for spotting the issue with the State information. I've now adjusted the code to reduce the items displayed for the State. I also replaced the uid with user or operator in the 'CANCELLED by' string.

**Review by bedroge — APPROVED** [review]

**Pull request merged**